src/glyph/parse.rs

```diff
@@ -532,6 +536,73 @@ fn start(reader: &mut Reader<&[u8]>, buf: &mut Vec<u8>) -> Result<GlyphBuilder,
     }
 }
 
+/// Move libs from the lib's `public.objectLibs` into the actual objects.
+/// They key will be removed from the glyph lib.
+fn fill_in_libs(glyph: &mut Glyph) -> Result<(), Error> {
+    if let Some(glyph_lib) = &mut glyph.lib {
+        if let Some(object_libs) = glyph_lib.remove("public.objectLibs") {
```

Comment:
I would write this as,

```rust
    let object_libs = match glyph.lib.as_mut().and_then(|lib| lib.remove("public.objectLibes")) {
        Some(lib) => lib,
        None => return Ok(()),
    };
```

And then you save yourself two layers of nesting, below.